src/app/pages/credentials/new-users/all-users/user-details/user-access-card/user-access-card.component.html

```diff
@@ -28,6 +28,9 @@ <h3 mat-card-title>
             {{ 'No Password' | translate }}
           } @else {
             {{ 'Has Password' | translate }}
+            @if (user.password_change_required) {
+              <span>{{ ' - (Change Required)' | translate }}</span>
```

Comment:
we can improve it to avoid similar/duplicated translations.
<img width="335" alt="Screenshot 2025-05-06 at 09 36 38" src="https://github.com/user-attachments/assets/be63e0f5-bc44-440c-937a-8bebf71c8895" />